yt_dlp/extractor/livestreamfails.py

```diff
@@ -3,21 +3,27 @@
 
 
 class LivestreamfailsIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?livestreamfails\.com/clip/(?P<id>[0-9]+)'
-    _TESTS = [{
-        'url': 'https://livestreamfails.com/clip/139200',
-        'md5': '8a03aea1a46e94a05af6410337463102',
-        'info_dict': {
-            'id': '139200',
-            'ext': 'mp4',
-            'display_id': 'ConcernedLitigiousSalmonPeteZaroll-O8yo9W2L8OZEKhV2',
-            'title': 'Streamer jumps off a trampoline at full speed',
-            'creator': 'paradeev1ch',
-            'thumbnail': r're:^https?://.+',
-            'timestamp': 1656271785,
-            'upload_date': '20220626',
+    _VALID_URL = r'https?://(?:www\.)?livestreamfails\.com/(clip|post)/(?P<id>[0-9]+)'
```

Comment:
Unused groups should be non capturing.
```suggestion
    _VALID_URL = r'https?://(?:www\.)?livestreamfails\.com/(?:clip|post)/(?P<id>[0-9]+)'
```